container/repository.go

```diff
@@ -0,0 +1,22 @@
+package container
+
+import (
+	"context"
+
+	"github.com/utahta/momoclo-channel/adapter/persistence"
+	"github.com/utahta/momoclo-channel/domain/entity"
+	"github.com/utahta/momoclo-channel/infrastructure/dao"
+)
+
+type repositoryContainer struct {
+	ctx context.Context
+}
+
+// Repository returns repositories container
+func Repository(ctx context.Context) *repositoryContainer {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported func Repository returns unexported type *container.repositoryContainer, which can be annoying to use